tests/src/JIT/HardwareIntrinsics/Arm64/Crypto.cs

```diff
@@ -0,0 +1,331 @@
+using System;
+using System.Diagnostics;
+using System.Runtime.CompilerServices;
+using System.Runtime.InteropServices;
+using System.Runtime.Intrinsics;
+using System.Runtime.Intrinsics.Arm.Arm64;
+
```

Comment:
@debayang Building these tests will fail until API is added to corefx.

If we want to merge this, it will need `#if`.  Something like I did here https://github.com/dotnet/coreclr/blob/master/tests/src/JIT/HardwareIntrinsics/Arm64/Simd.cs#L6

Reply:
Thanks @sdmaclea . 
Fixed